shishito/runtime/shishito_support.py

```diff
@@ -93,8 +104,9 @@ def gid(self, key, section=None):
             return value
 
         for cfg, cfg_name in self.configs:
-            if key in cfg and cfg[key] != '':
-                return cfg[key]
+            value = cfg.get(key)
+            if value:
```

Comment:
this is rare place where you maybe wants distinguish missing value and empty value
depends how you want to handle empty string
so it can be valid here 
`if value is not None:`